react/components/CustomHeader.tsx

```diff
@@ -30,8 +30,11 @@ const CustomHeaderLayout = ({ device }: { device: Device }) => {
 const CustomHeader: FunctionComponent = () => {
   const { isMobile } = useDevice()
 
-  return (
-    <CustomHeaderLayout device={isMobile ? Device.mobile : Device.desktop} />
+  return useMemo(
+    () => (
+      <CustomHeaderLayout device={isMobile ? Device.mobile : Device.desktop} />
```

Comment:
you could wrap CustomHeaderLayout into a memo, or not? But whatever

Reply:
☝️ thiss

Use `React.memo` instead